src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/WorkflowScheduler.kt

```diff
@@ -0,0 +1,64 @@
+package com.cognifide.gradle.aem.common.instance.service.workflow
+
+import com.cognifide.gradle.aem.common.instance.service.repository.Node
+import com.cognifide.gradle.aem.common.instance.service.repository.ResourceType
+import org.apache.http.HttpStatus
+
+class WorkflowScheduler(val workflow: Workflow) {
+
+    private val instance = workflow.instance
+    private val logger = workflow.logger
+
+    fun schedule(nodes: Sequence<Node>): Int {
+        nodes.forEach { scheduleForNode(it) }
+        return nodes.count()
+    }
+
+    fun schedule(path: String, type: ResourceType): Int {
+        val nodes = queryNodes(path, type)
+        return schedule(nodes)
+    }
+
+    fun scheduleForNode(node: Node) {
+
+        val params = mapOf(
+            "payload" to node.path,
+            "model" to workflow.model.path,
+            "payloadType" to "JCR_PATH"
+        )
+
+        instance.sync.http {
+            post(INSTANCE_URI, params) {
+                if (it.statusLine.statusCode != HttpStatus.SC_CREATED) {
+                    throw WorkflowException(
+                        "Workflow scheduling failed for ${params.get("payload")} and workflow model: $workflow" +
```

Comment:
please wrap `${params.get("payload")}` into `'` 

`and workflow model` => `and model`

DRY rule